client/src/Helpers/handle-channel-updates.ts

```diff
@@ -111,10 +113,25 @@ export const handleNewMessageData = (
       channel.messages = []
     }
     channel.messages.unshift(payload)
-  } else {
+  } else if (payload.messageType === CUSTOM_MESSAGE) {
     handleNonInfoMessage(res, payload.details.channel, payload, playerId)
+  } else {
+    const messageType = get(payload, 'messageType')
```

Comment:
@lebaphi  - can we please keep knowledge of deltas, and how to create/merge them inside `core-mapping`?

Reply:
To do the above, I _think_ we just call `handleNonInfoMessage` (as before), and then handle the new `messages` in `CoreMapping`

Reply:
I see, I will try this 👍